terraform/aws/environments/dev/cluster/variables.tf

```diff
@@ -0,0 +1,57 @@
+
+variable "vpc_id" {
+    default = ""
+    type    = "string"
+
+}
+
+variable "subnet_ids" {
+    default = ["", ""]
```

Comment:
nit: I am curious, is this default any different than `[""]`?

Reply:
Nope no difference. It was copied this way because I was testing with two subnets. Nice catch, removed ;)